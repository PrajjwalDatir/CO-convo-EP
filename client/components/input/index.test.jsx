import React from 'react';
import { mount } from 'enzyme';
import * as R from 'ramda';
import Input from './index';

describe('<Input />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<Input value="" onChange={R.always()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
