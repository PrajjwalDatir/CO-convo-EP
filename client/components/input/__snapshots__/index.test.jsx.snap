// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> should match snapshot 1`] = `
<ForwardRef
  className=""
  onChange={[Function]}
  value=""
>
  <input
    className="rounded-lg bg-gray-200 px-3 py-2 border border-solid border-transparent outline-none hover:border-gray-400 focus:bg-white focus:border-gray-400 "
    onChange={[Function]}
    value=""
  />
</ForwardRef>
`;
